packages/sandbox/ui/src/components/component-controls/PropsItem.tsx

```diff
@@ -65,6 +65,7 @@ export const PropsItem = pureComponent(
           {isCheckboxValues(possibleValues)
             ? (
               <ValueCheckbox
+                propPossibleValues={possibleValues}
```

Comment:
**General discussion**
So to make it work later, all we have to do in meta files is to change this:
```
isChecked: [true],
```
to this?
```
isChecked: [false, true]
```

Reply:
Exactly 👍

Reply:
It didn't work for me in `@primitives/checkbox/meta.txs`. Is it because we're using sandbox from nextools in this repo? Should I test this change in our target repo? 😅

Reply:
😅 i had to testPublish it each time with this branch on bubble-ui: `try-sandbox` 😞

Reply:
Another general question. Would it be possible and would it make sense to make the props that follow this anti pattern to only accept `false` and not even show them in code preview if we pass `true`? 🤔

Reply:
Ok, I think it makes sense in the `isChecked` case 🤔 Nevermind then. I just added new comment instead of removing previous one to not confuse you, because I'm sure you'll get emails about each one of them 😅